### PR TITLE
Feature/gifs - Added Ability for Geoff to Post Gifs!

### PR DESCRIPTION
**Pull request by tylermainguy**
New command **gif** allows Geoff to post gifs to the chat. Uses the GIPHY Core SDK wrapper on GIPHY API client to fetch gifs, and randomly selects one to display. API key associated with the project and stored in .env. Only added requirement is the giphy_client module.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
 .DS_STORE
 .vscode
 .env
-__pycache__
\ No newline at end of file
+__pycache__.idea/
```
Comment (joeytepp) [inline .gitignore]: Can we move this onto a new line?

File: bot.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 import re
 import time
 from commands import help_me
+from commands import get_gif
```
Comment (joeytepp) [inline bot.py]: Nice job keeping things modular! Can we change this to
```python
from commands import help_me, get_gif
```
so it fits on one line?

File: bot.py [hunk 3]
```diff
@@ -35,13 +40,16 @@ def parse_direct_mention(message_text):
 
 def handle_command(command, channel):
     # Default response is help text for the user
-    default_response = f"Howdy :sad_cowboy:! I'm geoff, the QMIND Bot that makes your lives easier. Try me out by using a command like *{HELP_COMMAND}*!"
+    default_response = f"Howdy :sad_cowboy:! I'm geoff, the QMIND Bot that makes your lives easier. Try me out by using a command like *{HELP_COMMAND} or {GIF_COMMAND}*!"
```
Comment (joeytepp) [inline bot.py]: Awesome! Small nitpick but could we keep the "or" unbolded? So just stars around the individual commands?

File: bot.py [hunk 4]
```diff
@@ -12,9 +13,13 @@
 slack_client = SlackClient(os.environ.get('BOT_OAUTH_TOKEN'))
 starterbot_id = None
 
+#Get API key for giphy
+giphy_api_key = os.environ.get("BOT_GIPHY_API_KEY")
```
Comment (joeytepp) [inline bot.py:16]: Great job using this as an environment variable! Could we update the `.env.example` file as well?

**Review by joeytepp — COMMENTED** [review]
This is looking dope! A few small comments to address then we should be good to go 🚢

File: bot.py (hunk 4 above)
Comment (tylermainguy) [reply bot.py:16]: Not sure what the format of .env.example is (current file is just empty for me)

Comment (joeytepp) [reply bot.py:16]: I'm gonna make a quick PR before I re-deploy, so I'll address it there

**Review by joeytepp — APPROVED** [review]
Great job! Fired up to keep improving geoff 🚀

**Pull request merged**